bin/server_start.sh

```diff
@@ -28,7 +28,7 @@ GC_OPTS="-XX:+UseConcMarkSweepGC
 # To truly enable JMX in AWS and other containerized environments, also need to set
 # -Djava.rmi.server.hostname equal to the hostname in that environment.  This is specific
 # depending on AWS vs GCE etc.
-JAVA_OPTS="-XX:MaxDirectMemorySize=$MAX_DIRECT_MEMORY \
+JAVA_OPTS="-XX:MaxDirectMemorySize=512M \
```

Comment:
Hmm.. this change seems to take out a PR that we merged recently.... we want this $MAX_DIRECT_MEMORY there so users can configure it, can we revert this change? Thanks.